lib/rules/sort-character-class-elements.ts

```diff
@@ -37,9 +46,50 @@ function getCharacterClassElementKind(
             ? "\\s"
             : "\\p"
     }
+    if (node.type === "ClassStringDisjunction") {
+        return "\\q"
+    }
+    if (
+        node.type === "CharacterClass" ||
+        node.type === "ExpressionCharacterClass"
+    ) {
+        return "[]"
+    }
     return "*"
 }
 
+/**
+ * Return the lexicographically smallest string accepted by the given element.
+ * If the class set is negate, the original value is used for calculation.
+ */
+function getLexicographicallySmallestFromElement(
+    node: CharacterClassElement,
+    flags: ReadonlyFlags,
+): ReadonlyWord {
+    const us =
+        node.type === "CharacterSet" && node.negate
+            ? toUnicodeSet({ ...node, negate: false }, flags)
+            : toUnicodeSet(node, flags)
+    const minimumWords: ReadonlyWord[] = [
+        ...(us.chars.isEmpty ? [] : [[us.chars.ranges[0].min]]),
+        ...us.accept.words,
```

Comment:
Uhhhmmm, my bad. `StringSet#words` wasn't meant to be public, and I made it private in the last update. The code still technically works, because the field is still there, but we really shouldn't rely on it, because I will probably remove it in the future.

Reply:
Thanks for fixing it!